rbac/management/group/view.py

```diff
@@ -225,6 +225,17 @@ def protect_system_groups(self, action, group=None):
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
 
+    def restrict_custom_default_group_renaming(self, request, group):
```

Comment:
This method does multiple  things:
- check HTTP method
- check custom group
- raise Validation error 

Is there a change to split it ? More details in comments

Can this logic be part only of update operation ? (So we don't need check on  HTTP method)